### DISCUSSION TITLE
Loding Skeleton component

### DISCUSSION THREAD

**Discussion by ryadav96** [heroui-inc/heroui#873] [Feature requests]
It will be very helpful if we implement skeleton component.
**Like :** _[https://chakra-ui.com/docs/components/skeleton]([Skeleton](https://chakra-ui.com/docs/components/skeleton))_

**Comment by jrgarciadev** [comment]
Hey please update to v2.0 https://nextui.org/blog/nextui-v2.

https://nextui.org/docs/components/skeleton
